modules/single/hash_demulti/demuxem.nf

```diff
@@ -5,6 +5,8 @@ process demuxem{
     publishDir "$projectDir/$params.outdir/$params.mode/hash_demulti/demuxem", mode:'copy'
     label 'small_mem'
     
+    conda "python=3.9 bioconda::pegasuspy 'pandas<2.0.0' demuxEM"
```

Comment:
Is it a lot of work to make this pandas 2.x compatible? That'd be great together with increasing the Python version to 3.11